monai/transforms/utils.py

```diff
@@ -642,6 +643,63 @@ def generate_label_classes_crop_centers(
     return ensure_tuple(centers)
 
 
+def combineMCtIandGLCCC(
```

Comment:
Would be better to use a more meaningful name, such as `map_and_generate_sampling_centers`.

Reply:
I would second here that we need a much more descriptive name here. It would also help to state the dimensions input tensors must have and what the dimensions mean in the docstring. Thanks!

Reply:
I'll change the name, Thanks!